src/test/resources/it/requests/gocardless/create-payment.json

```diff
@@ -3,7 +3,7 @@
     "amount": {{amount}},
```

Comment:
Missing quotes "{{amount}"

Reply:
yes, it is integer and github does not understand it. It's not a super nice thing what we're doing, but it works